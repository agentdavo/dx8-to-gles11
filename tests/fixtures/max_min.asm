max r0, v0, v1
min r0, v0, v1
