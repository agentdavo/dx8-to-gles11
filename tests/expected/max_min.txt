TEX_ENV_COMBINE func=32776
TEX_ENV_COMBINE func=32775
